### PR TITLE
Fix ContentPost

### PR DESCRIPTION
**Pull request by tfussell**
Fixes https://github.com/venturemark/webclient/issues/455
Screen Shot 2021-12-09 at 6 09 14 PM

### CODE REVIEW

File: src/component/contentpost/index.tsx [hunk 1]
```diff
@@ -155,42 +163,20 @@ function ContentPost(props: ContentPostProps) {
...
-            } catch (error) {}
-          }
-          return (
-            <div style={{ fontSize: "18px", fontWeight: 400, marginTop: 0 }}>
-              {title}
-            </div>
-          );
-        },
-      }}
+      title={title}
```
Comment (tfussell) [inline src/component/contentpost/index.tsx:166]: Prevents overriding of the intended Plasmic styles.

File: src/component/contentpost/index.tsx [hunk 2]
```diff
@@ -155,42 +163,20 @@ function ContentPost(props: ContentPostProps) {
...
       description={{
         as: ReadonlyEditor,
         props: {
           editor,
           editorShape,
         },
-        wrap(node) {
```
Comment (tfussell) [inline src/component/contentpost/index.tsx:180]: Superseded by new `hasDescription` property.

File: src/component/contentpost/index.tsx [hunk 3]
```diff
@@ -155,42 +163,20 @@ function ContentPost(props: ContentPostProps) {
...
       }}
       hasMedia={!!update?.image}
+      hasDescription={!!update?.text}
       actionMedia={{
         imageSource: update?.image || "",
         isEdit: false,
       }}
-      textContainer2={{
-        style: {
-          width: "100%",
```
Comment (tfussell) [inline src/component/contentpost/index.tsx:191]: No longer needed (as far as I can tell).

**Comment by peterphan1996** [comment]
LGTM

**Pull request merged**